### ISSUE TITLE
[Bug] vnode files gives an error

### ISSUE THREAD

**Issue by harmbuisman** [vantage6/vantage6#883]
**Describe the bug**
I am upgrading to vantage6 4.0. vnode start gives me the error "Exception: No configurations could be found!"

When I then run vnode files I get the same error.

**Expected behavior**
I would expect vnode files to give the directory configuration files should be placed, not give an error if there are no configurations yet.

**Comment by bartvanb** [comment]
Version 3 and 4 are not compatible. In this case, configuration files from v3 are not shown I think because they are no longer usable in v4 (e.g. because environments are no longer supported). You should create a new configuration file, or manually adapt it to match the template config in the [docs](https://docs.vantage6.ai/en/main/node/configure.html#all-configuration-options).

**Comment by harmbuisman** [comment]
Fine that they are not compatible, but that does not address the remark and expected outcome of the command "vnode files" to give me the folder where the configuration files are expected to be found.

**Comment by bartvanb** [comment]
Ah right, I think I didn't read the expected behavior properly. However, I disagree with what you expect: `vnode files` is meant to print files for an existing node configuration, so we can't do that if no node configurations exist.

We could print both an error and a message that says which directory we scanned for node configurations, would that help? 

Note that the directories where configuration files are stored are also listed in the docs, but it may be helpful to print them also here.

**Comment by bartvanb** [comment]
Putting this on hold: the algorithm store branch should be merged first, as this requires a change in the `AppContext` classes which have been moved on that branch.

**Comment by bartvanb** [comment]
Duplicate/related? #1098
